core/shared/src/main/scala/zio/internal/FiberContext.scala

```diff
@@ -38,7 +39,7 @@ private[zio] final class FiberContext[E, A](
   startSStatus: SuperviseStatus,
   parentTrace: Option[ZTrace],
   initialTracingStatus: Boolean,
-  fiberRefLocals: FiberRefLocals
+  private[zio] val fiberRefLocals: FiberRefLocals
```

Comment:
If this is a global variable, as it looks, we should just make it a singleton to avoid increasing memory consumption by the cost of a reference (per fiber).

Reply:
Just is just a field of `FiberContext` as it was before (only `private[zio] val` was added to make it accessible from outside `FiberContext`)